shimmer/docs/iota-sdk/docs/contribute.md

```diff
@@ -0,0 +1,49 @@
+---
+description: Contribute to the IOTA SDK joining the IOTA Libraries Initiative, contributing to the official GitHub repository or sharing your knowledge on Discord.
+keywords:
+  - join
+  - documentation
+  - project
+  - contribute
+  - discord
+  - GitHub
+---
+
+# Contribute to the IOTA SDK
+
+If you're thinking about contributing to the IOTA SDK project, please consider the following ways to get involved.
+
+## Join the IOTA Libraries Initiative
+
+The [IOTA Libraries Initiative](https://github.com/iota-community/X-Team_IOTA_Libraries) is a collaborative effort aimed
+at enhancing the developer experience. You can contribute in the following areas:
+
+- Quality assurance and code review
+- Documentation improvements
+- Code sample creation
+
+To get started, join the #experience channel on [Discord](https://discord.iota.org).
+
+## Contribute to the Project's GitHub Repository
+
+All of the project's code is open source and hosted on [GitHub](https://github.com/iotaledger/iota-sdk). You can contribute
+in the following ways:
+
+- Report any bugs you encounter
+- Suggest new features or enhancements
+- Contribute to the project's documentation
+
+## Contribute to the Documentation
+
+The project's documentation is also open source and available on GitHub. If you wish to contribute new documentation or
+fix any errors, please refer to
+the [contribution guidelines](https://github.com/iotaledger/documentation/blob/develop/.github/CONTRIBUTING.md).
+
+## Share Your Knowledge
+
+Helping others is a crucial aspect of any open source ecosystem. By sharing your knowledge with the community, you can
+provide immense value and potentially inspire others to learn and contribute. Join the ongoing discussions in the
+#clients-discussion channel on [Discord](https://discord.iota.org).
```

Comment:
if we created a new channel we should probably update this

Reply:
```suggestion
#libraries channel on [Discord](https://discord.iota.org).
```